docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -527,14 +521,15 @@ To enable the generation of OpenShift resources, you need to include OpenShift i
 quarkus.kubernetes.deployment-target=openshift
 ----
 
-If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (coma separated).
+If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (comma separated).
 
 [source]
 ----
 quarkus.kubernetes.deployment-target=kubernetes, openshift
 ----
 
-Note: In latest versions of Quarkus a wrapper extension of link:openshift[Openshift] is provided. If you use it, there is no need to explicitly set this property.
+NOTE: Quarkus also provides the link:openshift[Openshift] extension. This extension is basically wrapper around the Kubernetes extension and
+relieves Openshift users of the necessity of setting the `deployment-target` property to `openshift`
```

Comment:
Could you maybe check the rest of the doc to be sure the case is right?

Reply:
Done!